crates/bevy_dev_tools/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+#![warn(missing_docs)]
```

Comment:
```suggestion
```

This can be removed. `missing_docs` is now warn-by-default.

https://github.com/bevyengine/bevy/blob/f5ab1040a50edfe6f93791c28014915f49ea1933/Cargo.toml#L47-L49